### DISCUSSION TITLE
Tracking Usage of ArcGIS Assistant via Portal History

### DISCUSSION THREAD

**Discussion by knoopum** [EsriPS/arcgis-assistant-feedback#64] [Q&A] [answered]
I am interested in tracking the usage of ArcGIS Assistant in our ArcGIS Online organization. In looking at our org's [Portal History](https://developers.arcgis.com/rest/users-groups-and-items/portal-history.htm) logs, it looks like the appId for ArcGIS Assistant is D8cFRXEk41FR47VD. 

Can you confirm that is the correct appId? 

Also, is it safe to assume that you do not plan to change the appId going forward? If you do need to change the appId for some reason, then perhaps you could publicize the change? 

Thanks!

**Comment by jpeterson** [comment, marked as answer]
Hey @knoopum. Yep, that is the correct Client ID.

There is no plan to change the Client ID, but we can't guarantee it won't happen. I've made a note to share any such update in our changelog and on this discussion thread if it does happen.
